### ISSUE TITLE
Separate travis tests against devel of arangodb?

### ISSUE THREAD

**Issue by F21** [arangodb/arangodb-php#66]
I am building and testing this driver against the devel of arangodb (because I am planning to use the latest features in one of my apps).

When running phpunit, I am getting a few tests that consistently fails, while the current travis tests against 1.2.0 passes fine.

I propose that we have a totally separate test against the devel or arangodb. This means we will have 2 build status images.

The benefit of this is that sometimes, when things like the statistics are disabled, the statistics tests will fail. And we can just leave it alone in the failing state.

However, I am also seeing `GraphExtendedTest::testSaveVerticesAndSaveReplaceUpdateAndRemoveEdge` and a few others failing, so we can quickly fix those.

So, in other words, we have a travis for the latest stable version of arangodb, and a travis for the bleeding-edge devel. If the bleeding edge devel fails, it is not a really big issue due to things like statistics being disabled. But if we find something we can fix immediately, then that is good because, now, potential bugs that wait until an arangodb release is cut down the road is now fixed immediately.

Let me know what you guys think :)
